proto/internalapi/central/runtime_filters.proto

```diff
@@ -0,0 +1,13 @@
+syntax = "proto3";
+
+package central;
+
+import "storage/resource_collection.proto";
+import "storage/runtime_filters.proto";
+
+option go_package = "./;central";
+
+message RuntimeFilteringConfiguration {
```

Comment:
This should probably be defined in storage so both Central and Sensor can send this message.

Reply:
Done